### PR TITLE
In ruby

### LINKED ISSUE #1: Try rewriting in ruby
Octokit seems to be the thing:
* https://rubygems.org/gems/octokit/versions/4.21.0
* https://octokit.github.io/octokit.rb/

Here are some similar projects:
* https://github.com/deis/seed-repo
* https://github.com/bond-agency/github-label-maker
* https://github.com/izuzak/github_labeler

The main change I want to make is to keep labels organized by category, with the color assigned to the category rather than the label.

### PR DESCRIPTION
**Pull request by hackartisan**
closes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,12 @@
 # dls-github-label-maker
 apply a standardized set of labels to github repositories
+
+# Run instructions
+To apply all the labels from labels.json to a repository, do:
+
+`$ bin/labeler apply_labels [organization/repository]`
+
+These scripts expect you to have a Github personal access token exported in your shell, under the name `GITHUBTOKEN`. That token should have `repo` access. For more information about creating a Github token, check out [their docs](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/).
```
Comment (tpendragon) [inline README.md:9]: Can we pull pulbot's token via lastpass for this?

File: Gemfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+source "https://rubygems.org"
+
+git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
+
+gem "octokit", "~> 4.0"
+gem "pry"
+gem "thor"
+
+group :test do
+  gem "rspec"
```
Comment (tpendragon) [inline Gemfile:12]: simple_cov?

**Review by tpendragon — CHANGES_REQUESTED** [review]
The code looks great! Just a couple small suggestions.

**Comment by hackartisan** [comment]
@tpendragon what would you think about creating issues for both of those, just to keep this PR from getting too big?

**Review by tpendragon — APPROVED** [review]

**Pull request merged**